src/load_dataset.py

```diff
@@ -79,6 +85,9 @@ def create_bigquery_objects(client, project, dataset):
                      bigquery.SchemaField("cas_feature_index", "INTEGER", mode="REQUIRED"),
                      bigquery.SchemaField("original_feature_id", "STRING", mode="REQUIRED"),
                      bigquery.SchemaField("feature_name", "STRING", mode="REQUIRED"),
+                     bigquery.SchemaField("var_metadata", "STRING", mode="REQUIRED"),
+                     # TODO get direct JSON metadata loading working
```

Comment:
Is there a ticket for this?

Reply:
No but there should be. Kristian and I talked about it briefly in Slack on Friday afternoon and I had hoped to roll it into this PR but that didn't work out.

Reply:
VS-546 created.